engine/src/test/java/com/datatorrent/stram/plan/logical/LogicalPlanConfigurationTest.java

```diff
@@ -676,18 +659,22 @@ public void populateDAG(DAG dag, Configuration conf)
     dagBuilder.prepareDAG(dag, app, appName);
     LogicalPlan.OperatorMeta ometa = dag.getOperatorMeta("TestModule$DummyOperator");
     LogicalPlan.OperatorMeta om = null;
-    for (Map.Entry<LogicalPlan.OutputPortMeta, LogicalPlan.StreamMeta> entry : ometa.getOutputStreams().entrySet()) {
-      if (entry.getKey().getPortName().equals("output")) {
-        om = entry.getKey().getUnifierMeta();
-      }
-    }
 
     /*
      * Verify the attribute value after preparing DAG.
      */
-    Assert.assertNotNull(om);
-    Assert.assertEquals("", Integer.valueOf(2), om.getValue(Context.OperatorContext.TIMEOUT_WINDOW_COUNT));
+    for (OutputPortMeta portMeta : ometa.getOutputStreams().keySet()) {
+      if (portMeta.getPortName().equals("output")) {
+        om = portMeta.getUnifierMeta();
+        Assert.assertNotNull(om);
```

Comment:
It was added to have precise check if unifierMeta is null. I agree it would be detected on next statement but with extra stack of null pointer exceptions rather than intended check is for unifier count. Removed it.